web/app.php

```diff
@@ -1,16 +1,19 @@
 <?php
 
-use Symfony\Component\ClassLoader\ApcClassLoader;
 use Symfony\Component\HttpFoundation\Request;
 
-$loader = require_once __DIR__.'/../app/bootstrap.php.cache';
+/**
+ * @var Composer\Autoload\ClassLoader $loader
+ */
+$loader = require __DIR__.'/../app/autoload.php';
+include_once __DIR__.'/bootstrap.php.cache';
```

Comment:
we use include instead of require, as the app is even working if the file cannot be loaded.